api/trace.proto

```diff
@@ -52,3 +53,13 @@ message ZipkinConfig {
   // /api/v1/spans, which is the default value.
   string collector_endpoint = 2 [(validate.rules).string.min_bytes = 1];
 }
+
+message DynamicOtConfig {
```

Comment:
nit: Can you add a small message level description?

Reply:
Added.